### DISCUSSION TITLE
DataType change for SB1 (commit 49e6938)

### DISCUSSION THREAD

**Discussion by mareksm** [oracle/python-oracledb#245] [General]
Is there a reason why TNS_DATA_TYPE_SB1 representation was changed from TNS_TYPE_REP_UNIVERSAL to TNS_TYPE_REP_ORACLE? (https://github.com/oracle/python-oracledb/commit/49e6938472c76b7215db963a921e2bfdcca051e3)

**Comment by anthony-tuininga** [comment]
Yes! It reduces the number of bytes sent since there is no need to send the length -- since the length would always be 0 or 1 anyway! It also paves the way for another enhancement which is being considered internally currently.

**Comment by mareksm** [comment]
Thank you for the update.
